installer/generate_secrets.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+from base64 import b64encode
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+import json
+import os
+import secrets
+
+
+def input_field(component, name, description):
+  prompt_string = f"[{component} {name}] ({description}): "
+  return input(prompt_string)
+
+def generate_tap_secrets():
+  fname = input_field("TAP", "google_creds.json",
+    "file containing google service account credentials")
+
+  with open(fname, 'r') as f:
+    return {
+      "slack_webhook_url": input_field("TAP", "slack_webhook_url",
+        "slack webhook url for querymonkey"),
+      "google_creds.json": b64encode(f.read().encode("utf-8")).decode("utf-8")
```

Comment:
Nit: `utf-8` will always be the default so you can just use `.encode()` and `.decode()`.